rustls/src/crypto/ring/mod.rs

```diff
@@ -2,7 +2,8 @@ use crate::crypto::{CryptoProvider, SupportedKxGroup};
 use crate::rand::GetRandomFailed;
 use crate::suites::SupportedCipherSuite;
 
-use ring::rand::{SecureRandom, SystemRandom};
+pub use ring as ringlike;
```

Comment:
Nit: I feel like calling it `ring_like` would be more aligned with the naming guidelines.